test/unit/uk/gov/hmrc/apinotificationpull/util/XmlBuilderSpec.scala

```diff
@@ -16,16 +16,25 @@
 
 package uk.gov.hmrc.apinotificationpull.util
 
+import org.mockito.Mockito._
+import org.scalatest.mockito.MockitoSugar
+import uk.gov.hmrc.apinotificationpull.config.AppContext
 import uk.gov.hmrc.apinotificationpull.model.Notifications
-import uk.gov.hmrc.apinotificationpull.util.XmlBuilder.toXml
 import uk.gov.hmrc.play.test.UnitSpec
 
-class XmlBuilderSpec extends UnitSpec {
+class XmlBuilderSpec extends UnitSpec with MockitoSugar {
+
+  trait Setup {
+    val apiContext = "notifications"
+    val appContext = mock[AppContext]
+    when(appContext.apiContext).thenReturn(apiContext)
+    val xmlBuilder = new XmlBuilder(appContext)
+  }
 
   "XmlBuilder.toXml()" should {
 
-    "convert notifications to XML" in {
-      val notifications = Notifications(List("/notifications/123", "/notifications/456"))
+    "convert notifications to XML" in new Setup {
+      val notifications = Notifications(List("/notification/123", "/notification/456"))
```

Comment:
are you sure of this change?

Reply:
yes, the path the queue service returns is `/notification/{notificationId}` we need to expose the path relative to this service by taking the `notificationId` and constructing the path with our api context. The change to this test replicates what we expect the notifications list to look like when retrieved from the queue.